content/millennium/r4/entities/organization.md

```diff
@@ -36,12 +36,13 @@ Search for Organizations that meet supplied query parameters:
 
 ### Parameters
 
- Name        | Required?                      | Type       | Description
--------------|--------------------------------|------------|----------------------------------------------------------
- `_id`       | This or `identifier` or `type` | [`token`]  | The logical resource id associated with the resource.
- `identifier`| This or `type` or `_id`        | [`token`]  | NPI, OID or FederalTAXID identifier for the organization.
- `type`      | This or `identifier` or `_id`  | [`token`]  | The organization's type. Example: `http://terminology.hl7.org/CodeSystem/organization-type|govt`
- [`_count`]  | No                             | [`number`] | The maximum number of results to return. Defaults to `100`.
+ Name        | Required?                               | Type       | Description
+-------------|-----------------------------------------|------------|----------------------------------------------------------
+ `_id`       | This or `identifier` or `type`          | [`token`]  | The logical resource id associated with the resource.
+ `identifier`| This or `type` or `_id`                 | [`token`]  | NPI, OID or FederalTAXID identifier for the organization.
+ `type`      | This or `identifier` or `_id`           | [`token`]  | The organization's type. Example: `http://terminology.hl7.org/CodeSystem/organization-type|govt`
+ [`_count`]  | No                                      | [`number`] | The maximum number of results to return. Defaults to `100`.
+ `name`      | This or `_id` or `identifier` or `type` | [`string`] | Name used for the organization.
```

Comment:
@NimishaMN  Do you have the description mentioned somewhere in jira. As per the [hl7](https://hl7.org/fhir/r4/organization.html) is should be _'A portion of the organization's name or alias'_

Reply:
Confirmed with SD (Alec) for the Description. Got a +1 that we can go ahead with 'Name used for the organization.'